### DISCUSSION TITLE
How to override theme style for inline raw code?

### DISCUSSION THREAD

**Discussion by blackjyn** [typst/typst#4001] [General]
So, I set the code theme using One Dark theme. But I want to override the inline raw text for its color fill. 
Well, it works for the rest of the text BUT not that raw text.

how do I accomplish that? 

```typ
#set raw(theme: "OneDark.tmTheme")
#show raw.where(block:false): it=>box(
 	fill: black,
  	inset: (x: 3pt, y: 0pt),
  	outset: (y: 3pt),
  	radius: 2pt,
    text(fill: red,ligatures: true,[#it will be red])
 )

The `text()`

```

rendered:

![typst-theme-iverride](https://github.com/typst/typst/assets/6046191/71d2d0b7-7b8e-4a09-a7b0-0a89bea9f7a9)

you see that the original raw text are kept using the One Dark theme and not overridden. 
Help?

**Comment by laurmaedje** [comment]
You could scope the original set rule to only affect block-level raw:
```typ
#show raw.where(block: true): set raw(theme: "OneDark.tmTheme")
```

**Comment by blackjyn** [comment]
thank you @laurmaedje ! 
I didn't expect it could be that simple solution.
